src/cmake/externalpackages.cmake

```diff
@@ -161,7 +160,7 @@ checked_find_package (Libheif VERSION_MIN 1.11
                       RECOMMEND_MIN_REASON "for orientation support")
 
 checked_find_package (LibRaw
-                      VERSION_MIN 0.20.0
+                      VERSION_MIN 0.21.3
```

Comment:
No, we don't want to do this. Changing it here means "we will not accept a version older than this." The minimum 0.20.0 was chosen purposefully.

Reply:
To explain this even better:

The versions in externalpackages.cmake express the oldest version we will accept. This is some kind of compromise between wanting it convenient for users who for various reasons are in an environment with versions of these packages that may be as much as a few years old, versus not wanting to have to require OIIO to have too much extra logic to cater to older APIs or lack of newer features in those dependencies. Also, we never raise minimum versions except at the annual release (like 2.4 -> 2.5), never at an intermediate patch (like 2.4.1 -> 2.4.2) which are guaranteed not to change build requirements.

The versions in build_foo.cmake or build_foo.bash express the version of the dependency that we will choose to download and build ourselves if an adequate one is not found in the environment. Those will tend to be fairly recent, though not necessarily the very latest one -- depends on how confident we feel that the latest has been thoroughly tested and is solid.

So in general, we expect the "oldest accepted" and "default for self-building" to be different from one another.

Reply:
While building from master I run into [recent changes not available in 0.20.0](https://github.com/AcademySoftwareFoundation/OpenImageIO/pull/4454/files) Had to use build_libraw.bash to get the updates.

Reply:
Ah, I see. I think that this means that we should have a `#if` around just those couple lines, like

    #if LIBRAW_VERSION >= LIBRAW_MAKE_VERSION(0, 21, 0)

And then in the docs where we describe `raw:max_raw_memory_mb`, just point out that it requires LibRaw >= 21 for it to be honored.

Can you add that to this PR, please, so then we don't have to raise the minimum libraw version just for that one feature? Thanks.

Reply:
>So in general, we expect the "oldest accepted" and "default for self-building" to be different from one another.

Thanks, for detailed explanation - that makes sense. 

>Can you add that to this PR  so then we don't have to raise the minimum libraw version

Of course, I'll add it.